admin/src/css/configuration.less

```diff
@@ -193,3 +193,17 @@ dl.horizontal {
   font-style: italic;
   color: @label-color;
 }
+
+#images-branding {
+  .navbar-inverse {
+    display: inline-block;
+    img {
+      height: 50px;
```

Comment:
Is it a good idea to force the size of the image? If somebody uploads a very large or very small image, it might be useful for them to see the original size? Unsure.

Reply:
I copied that from the application css so it appears the same in this preview as it does in the desktop app.